samples/valueentity-counter/src/main/java/com/example/Main.java

```diff
@@ -1,20 +1,29 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+
 package com.example;
 
 import com.akkaserverless.javasdk.AkkaServerless;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
-import static com.example.MainComponentRegistrations2.withGeneratedComponentsAdded;
+import com.example.domain.Counter;
+import static com.example.MainComponentRegistrations.registerAll;
 
 public final class Main {
-    
-    private static final Logger LOG = LoggerFactory.getLogger(Main.class);
-    
-    public static final AkkaServerless SERVICE =
-            withGeneratedComponentsAdded(new AkkaServerless());
-    
-    public static void main(String[] args) throws Exception {
-        LOG.info("starting the Akka Serverless service");
-        SERVICE.start().toCompletableFuture().get();
-    }
-}
+
+  private static final Logger LOG = LoggerFactory.getLogger(Main.class);
+
+  public static final AkkaServerless SERVICE =
+    // This withGeneratedComponentsAdded wrapper automatically registers any generated Actions, Views or Entities,
+    // and is kept up-to-date with any changes in your protobuf definitions.
+    // If you prefer, you may remove this wrapper and manually register these components.
+    registerAll(new AkkaServerless(),
+      Counter::new);
```

Comment:
I think we could simplify this with

```java
registerAll(Counter::new);
```

and let registerAll create an instance of `AkkaServerless` and return it.